src/Auth0Provider.cs

```diff
@@ -15,7 +17,7 @@ public class Auth0Provider : ITokenProvider
         private Auth0TokenResponse _cachedToken;
         private DateTime _expiry;
         private bool _isAuthRequired;
-
+        private readonly string _cachedTokenResponse = @"/tmp/tokenResponse.json";
```

Comment:
do we need to have a different path for different `Auth0TokenRequest`?

Reply:
tmp folder is unique to each lambda function so I don't think there's a need to create different path for different Auth0TokenRequest https://forums.aws.amazon.com/thread.jspa?messageID=610055

Reply:
What happens if the service needs to retrieve multiple tokens? In that case we need to do `/tmp/tokens_for_svc_1`, `/tmp/tokens_for_svc_2`. We can solve that by using `/tmp/{Audience}`